storybookjs/storybook
# Angular - prioritise template over generated source

Issue:

## What I did

Moved the template check above the source check. If a template exists then use it in the "Show code" block. Also prevents the computesTemplateSourceFromComponent from running unnecessarily

## How to test

Create a story with a template that differs from generated source. View "Show code" in docs and the template should display.
